sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -324,6 +324,34 @@ See the [Bash manual on file operators](https://tldp.org/LDP/abs/html/fto.html)
 
 Alternate suggestions include using `grep -c` to search for a valid string match, or other tool which will appropriately error when the expected output is not successfully created.
 
+### Stubs
+
+#### Stub block must exist
+
+A stub block MUST exist for all modules
+
+#### Stub files for all output channels 
+
+The stub block MUST include the creation of at least one file for every output channel (both mandatory and optional), generated with touch, e.g.
+
+```bash
+touch ${prefix}.txt
+```
```

Comment:
I would add that the sub block should reproduce as much as possible the number and filenames that are expected as output

Reply:
Done